overlord/devicestate/handlers_install.go

```diff
@@ -111,6 +113,17 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("cannot make run system bootable: %v", err)
 	}
 
+	// disable cloud-init by default (as it's not confined)
```

Comment:
if possible this should be done before making the run system bootable,  also I would put this into a helper configureRunSystem

Reply:
Indeed, we talked about this and I forgot already. Updated now.